commit 05a13bff22121bacffa0e5caf4459f67992a3517 (refs/changes/23/1823/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-04-25T14:11:57-07:00 (2 years, 5 months ago)
    
    DOCKER-1020 Define "public" network using Docker label
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/common/validation.js b/lib/common/validation.js
index 374db7c..613e5b8 100644
--- a/lib/common/validation.js
+++ b/lib/common/validation.js
@@ -31,7 +31,8 @@ var UUID_RE = /^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$/;
 var ALIAS_RE = /^[a-zA-Z0-9][a-zA-Z0-9\_\.\-]*$/;
 var RAM_RE = /^0$|^([1-9][0-9]*$)/;
 var TRITON_TAG_ROOT_RE = /^triton\./;
-var TRITON_TAG_DEFAULT_RE = /^triton\.cns\.(?:services|disable|reverse_ptr)$/;
+/* JSSTYLED */
+var TRITON_TAG_DEFAULT_RE = /^triton\.(?:cns\.services|cns\.disable|cns\.reverse_ptr|network\.public)$/;
 /* JSSTYLED */
 var DOCKER_TAG_DEFAULT_RE = /^(?:sdc_docker$|docker:label:triton\.|docker:label:(?:com|io|org)\.docker(?:project)?\.)/;
 
